### ISSUE TITLE
[Feature Request] Display assigned tags at Note edit/view pane

### ISSUE THREAD

**Issue by archont00** [laurent22/joplin#469]
### Operating system

- Linux

### Application

- Desktop

### Feature request

There is no way to quickly check, which tags are currently assigned to a displayed note. The user has to click on Tags icon to see the list.

![screenshot from 2018-04-29 10-53-09](https://user-images.githubusercontent.com/937861/39404988-0ef68d22-4b9d-11e8-8ee6-4ec406933a5d.png)

![screenshot from 2018-04-29 10-53-14](https://user-images.githubusercontent.com/937861/39404993-219f99aa-4b9d-11e8-9807-ae85e0b58226.png)

It would be nice if the currently assigned tags are displayed below the note title and optionally also highlighted in the left pane list of tags.

**Comment by Abijeet** [comment]
Working on this from my side. Here are few screenshots.

![tags-joplin](https://user-images.githubusercontent.com/1685517/46591133-4c7ef180-cad6-11e8-9849-61f89842f0a2.png)

**Comment by xuhcc** [comment]
Is this feature ready? PR #893 was merged, but I don't see note tags in desktop app (v1.0.127).

**Comment by cable729** [comment]
That PR was abandoned.

**Comment by Abijeet** [comment]
Hey guys, I initially implemented this and it was merged, but the changes I made caused some major regressions (https://github.com/laurent22/joplin/issues/995) and hence it was rolled back.

I've been wanting to push a proper fix for it, but haven't had a lot of *time* on my hands.  

I'm hesitant to give a timeline on when I might be able to work on it since I've done that before, and this just takes a back-seat. If anyone wants to give this a shot, [here](https://github.com/laurent22/joplin/pull/1019) are some comments from Laurent regarding how it should be implemented.
